contrib/raftexample/main.go

```diff
@@ -34,8 +34,10 @@ func main() {
 	defer close(confChangeC)
 
 	// raft provides a commit stream for the proposals from the http api
-	commitC, errorC := newRaftNode(*id, strings.Split(*cluster, ","), *join, proposeC, confChangeC)
+	rc := newRaftNode(*id, strings.Split(*cluster, ","), *join, proposeC, confChangeC, false)
```

Comment:
this should stay the same too; the channel interface is important for keeping the interface small